### PR TITLE
Kommander 1.2 Release Notes

### PR DESCRIPTION
**Pull request by surdy**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-72473

## Description of changes being made
Added a note about airgapped environment support in Kommander 1.2

## Checklist
- [X] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [X] Test all commands and procedures, if applicable.
- [X] Create your PR against `staging`, not `master`. 
- [X] Update all links if you are moving a page.
- [X] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 1]
```diff
@@ -32,19 +32,18 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.17.x  |
 | **Default**        | 1.17.3  |
 
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
+# New Features and Capabilities 
+
+## Support for air-gapped environment 
 
-## Features/Improvements
+## Improved catalog handling
```
Comment (cmastr) [inline pages/ksphere/kommander/1.2/release-notes/index.md:39]: @juliangieseke we need to expand upon each of these new features and link to the topic that goes into more details. Can you help? Here's the goal:

https://docs.d2iq.com/mesosphere/dcos/2.1/release-notes/2.1.0/#new-features-and-capabilities

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 2]
```diff
@@ -32,52 +31,55 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.17.x  |
 | **Default**        | 1.17.3  |
 
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
+# New Features and Capabilities 
+
+
+## Support for air-gapped environment
```
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:37]:
```suggestion
## Support for air-gapped environments
```

File: pages/ksphere/kommander/1.2/release-notes/index.md (hunk 1 above)
Comment (samvantran) [reply pages/ksphere/kommander/1.2/release-notes/index.md:39]: This link is broken but we did the same thing for Kommander 1.1.0 https://github.com/mesosphere/dcos-docs-site/blob/staging/pages/ksphere/kommander/1.1/release-notes/index.md#new-features-and-capabilities-in-kommander-110

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 3]
```diff
@@ -32,52 +31,55 @@ Kommander provides a command center for all your cloud native management needs i
...
+Kommander demo env 
+Support network tunnels
+app template  
+DKP product licensing 
+Show Correct Addon Status 
+Network Policy 
+Limit ranger 
+Kubefed scaling 
+Create EKS/etc clusters 
+Easy import process for 3rd party clusters
```
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:57]: These should be bulleted

**Review by samvantran — COMMENTED** [review]
Couple comments but looks good so far as prep

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 4]
```diff
@@ -32,52 +31,55 @@ Kommander provides a command center for all your cloud native management needs i
...
 
-## Component Versions
+* Added SAML IDP
+* Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters
+* Added a setting to configure thresholds for resource warnings 
+* Attaches EKS clusters now can show metrics
+* Added Infrastructure provider management on global level
```
Comment (juliangieseke) [inline pages/ksphere/kommander/1.2/release-notes/index.md:63]: these should be move below something like "additional new feature" os something.

```suggestion
## Additional New Features
* Added SAML IDP
* Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters
* Added a setting to configure thresholds for resource warnings 
* Attaches EKS clusters now can show metrics
* Added Infrastructure provider management on global level
```

File: pages/ksphere/kommander/1.2/release-notes/index.md (hunk 1 above)
Comment (juliangieseke) [reply pages/ksphere/kommander/1.2/release-notes/index.md:39]: uh, not sure I got enough context on them 🙈 

Also, I think we need Product to decide which features we actually want to call out explicitly, not sure who did that for 1.1.0? @samvantran, so you know?

**Review by juliangieseke — COMMENTED** [review]
uh, these changes are super hard to read 🙈  this vale tool doesnt really help either.

nvm, some comments.

File: pages/ksphere/kommander/1.2/release-notes/index.md (hunk 3 above)
Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:57]: Sorry! this is still a WIP, I pulled this list from the TE slides. Gonna go through this with PMs.

File: pages/ksphere/kommander/1.2/release-notes/index.md (hunk 4 above)
Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:63]: Anything bulleted should go under fixed and improved issues. I moved these down.

File: pages/ksphere/kommander/1.2/release-notes/index.md (hunk 1 above)
Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:39]: GOOD glad we did this for 1.1.0. Not sure how that got lost as a format in 1.2. That link helps me reformat this page again.

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 5]
```diff
@@ -10,74 +9,54 @@ enterprise: false
...
-- kommander-licensing (yakcl): `0.4.2`
-- UI: `6.22.0`
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
\ No newline at end of file
+## Fixed and Improved Issues 
+* Improved Access Control for editing/deleting actions.
```
Comment (cmastr) [inline pages/ksphere/kommander/1.2/release-notes/index.md:48]: Do any of these have COPS ticket numbers or can we add links to additional information in the help?

Comment (surdy) [reply pages/ksphere/kommander/1.2/release-notes/index.md:48]: I could not find any by spot checking a few. I only found D2IQ tickets. The filter https://jira.d2iq.com/issues/?jql=project%20%3D%20%22Customer%20Operations%22%20%20%20AND%20(fixVersion%20%3D%20%22Kommander%201.2%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20Beta%200%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20Beta%201%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20RC%201%22%20%20%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20RC%202%22)%20 also did not result in anything related to these

**Review by surdy — COMMENTED** [review]
LGTM

**Review by cmastr — APPROVED** [review]

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 6]
```diff
@@ -10,74 +9,55 @@ enterprise: false
...
 
-**NOTE:** You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
-
-<!-- TBD when 1.2 goes GA:
-# Release Summary
+**NOTE:** You must be a registered user and logged on to the support portal to download this product. New customers must contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
 
+# Release Summary 
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated and cost management, across multiple clusters, whether they are a new Konvoy cluster or existing 3rd party/DIY distribution.
```
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:22]: I find the last sentence in this paragraph ("Additionally...") to be very confusing, I have no idea what 'unlock federated and cost management' means.   Suggested rewording:

```suggestion
Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
```

File: pages/ksphere/kommander/1.2/release-notes/index.md [hunk 7]
```diff
@@ -10,74 +9,55 @@ enterprise: false
...
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
\ No newline at end of file
+- Added SAML IDP.
+- Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters.
+- Added a setting to configure thresholds for resource warnings. 
+- Attached EKS clusters now show metrics.
```
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:54]: "One of these things is not like the others... "  :-) 

```suggestion
- Added the ability to show metrics in attached EKS clusters.
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Comment by cmastr** [comment]
moved all changes to #3301 and closing in favor of that PR.

**Pull request closed without merging**